src/routes/e2e/widget/inline/+page.svelte

```diff
@@ -2,7 +2,7 @@
   import { onMount } from 'svelte';
   import { page } from '$app/stores';
 
-  import Widget, { form, journey, user } from '$package/inline';
+  import Widget, { form, journey, user } from '$package/inline.svelte';
```

Comment:
another weird thing but i had to add this for stuff to work. it really didn't like this path

Reply:
Huh, interesting. This would definitely not be desired since this is the user facing package, and I wouldn't want our users to have to import something that looks like a Svelte component. Concerning ...

Reply:
the compiled file ends in `.svelte.js` so if we can remove the .svelte part we should be fine

Reply:
Then let's definitely look into how to remove the `.svelte` portion of the file name.

Reply:
from rollup docs:
```
This pattern will also be used for every file when setting the [output.preserveModules](https://rollupjs.org/guide/en/#outputpreservemodules) option. Note that in this case, [name] will include the relative path from the output root and possibly the original file extension if it was not one of .js, .jsx, .mjs, .cjs, .ts, .tsx, .mts, or .cts.
```

so i added a .replace on it. seemed to work, let me know if you don't like it though